machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,144 @@
+﻿#Set power scheme to high performance
+powercfg /S 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c
+#region GIT Install Config
+$gitInstallLog = @"
+[Setup]
+Lang=default
+Dir=C:\Program Files\Git
+Group=Git
+NoIcons=0
+SetupType=default
+Components=ext,ext\shellhere,ext\guihere,assoc,assoc_sh
+Tasks=
+PathOption=Cmd
+SSHOption=OpenSSH
+CRLFOption=CRLFAlways
+BashTerminalOption=ConHost
+PerformanceTweaksFSCache=Enabled
+UseCredentialManager=Enabled
+EnableSymlinks=Disabled
+"@
+#endregion
+$gitInstallLog > gitInstallInfo.log
+wget https://github.com/git-for-windows/git/releases/download/v2.12.0.windows.1/Git-2.12.0-64-bit.exe -OutFile git-2.12.exe
+$gitArgList = "/LOADINF=gitInstallInfo.log", "/VERYSILENT"
+echo Start-Process -FilePath .\git-2.12.exe -ArgumentList $gitArgList -Wait
+Start-Process -FilePath .\git-2.12.exe -ArgumentList $gitArgList -Wait
+
+wget https://cmake.org/files/v3.7/cmake-3.7.2-win64-x64.msi -OutFile cmake-3.7.2.msi
+$cmakeArgList = "/i", "cmake-3.7.2.msi", "/quiet", "/norestart", "ADD_CMAKE_TO_PATH=System"
+echo Start-Process -FilePath msiexec -ArgumentList $cmakeArgList -Wait
+Start-Process -FilePath msiexec -ArgumentList $cmakeArgList -Wait
+
+wget http://javadl.oracle.com/webapps/download/AutoDL?BundleId=218831_e9e7ea248e2c4826b92b3f075a80e441 -OutFile java.exe
+$javaArgList = ,"/s"
+echo Start-Process -FilePath java.exe -ArgumentList $javaArgList -Wait
+Start-Process -FilePath java.exe -ArgumentList $javaArgList -Wait
+
+wget https://www.python.org/ftp/python/2.7.13/python-2.7.13.msi -OutFile python27.msi
+$python27ArgList = "/i", "python27.msi", "/quiet", "/norestart", "ADDLOCAL=DefaultFeature,SharedCRT,Extensions,TclTk,Documentation,Tools,pip_feature,Testsuite,PrependPath"
+echo Start-Process -FilePath msiexec -ArgumentList $python27ArgList -Wait
+Start-Process -FilePath msiexec -ArgumentList $python27ArgList -Wait
+
+wget https://www.python.org/ftp/python/3.6.0/python-3.6.0.exe -OutFile python36.exe
+$python36ArgList = "/quiet", "PrependPath=1", "TargetDir=C:\Python36"
+echo Start-Process -FilePath python36.exe -ArgumentList $python36ArgList -Wait
+Start-Process -FilePath python36.exe -ArgumentList $python36ArgList -Wait
+
+wget https://benchviewstorage.blob.core.windows.net/public/vs_community.exe -OutFile vs_community.exe
+#region VS Admin File
+$vsAdminFile = @"
```

Comment:
Sure.  My initial desire was to make this an all-in-one script that you could just put on a machine, run and it would setup the machines.  But since we would probably just bring it down off of github having additional config files that come down as well should not be that hard.